apps/api/src/domain/models/playlist/queries/playlist-query-repository.interface.ts

```diff
@@ -27,4 +28,10 @@ export interface IPlaylistQueryRepository extends IPublishable, ICountable {
     allowUser(id: AggregateId, userId: AggregateId): Promise<void>;
 
     addAudioItem(id: AggregateId, audioItemId: AggregateId): Promise<void>;
+
+    translatePlaylistName(
+        id: AggregateId,
+        translation: String,
```

Comment:
I too am coming back to TS from Java, so I know where this is coming from. We should use `string` instead. This is because `string` is a primitive type in JavaScript (unlike Java), whereas `String` refers to the auto-boxed wrapper type. It's like `int` vs. `Integer` in Java. Long story short, let's stick to `string` in our types.